addon/components/frost-link.js

```diff
@@ -255,7 +255,9 @@ export default LinkComponent.extend(SpreadMixin, PropTypeMixin, {
     if (isEmpty(this.get('params'))) {
       let params = []
 
-      params.push(this.get('text'))
+      if (this.get('text')) {
```

Comment:
Can should probably store this.get('text') in a variable to avoid getting it twice:
```
const text = this.get('text')
if (text) {
  ...
}
```